src/test/regress/expected/replicate_reference_tables_to_coordinator.out

```diff
@@ -43,11 +43,13 @@ SELECT create_reference_table('numbers');
 
 INSERT INTO numbers VALUES (20), (21);
 NOTICE:  executing the command locally: INSERT INTO replicate_ref_to_coordinator.numbers_8000001 AS citus_table_alias (a) VALUES (20), (21)
+SET citus.enable_metadata_sync TO OFF;
```

Comment:
any reason for this change?

Reply:
Those functions are part of the c code and distributing them here cause having them on other test outputs.